presto-main/src/main/java/io/prestosql/operator/scalar/StringFunctions.java

```diff
@@ -899,4 +899,16 @@ public static Slice concat(@LiteralParameter("x") Long x, @SqlType("char(x)") Sl
 
         return result;
     }
+
+    @Description("Determine whether source starts with prefix or not")
+    @ScalarFunction("starts_with")
```

Comment:
nit: We can remove `("starts_with")` because the method name `startsWith ` will be translated to `starts_with` function.

Reply:
Removed.